components/molecules/HeaderLogo/header-logo.tsx

```diff
@@ -20,7 +20,7 @@ const HeaderLogo: React.FC<HeaderLogoProps> = ({ textIsBlack, withBg = false })
           height={32}
           src={withBg ? openSaucedImgWithBg : openSaucedImg}
         />
-        <p className={`font-bold !text-base ${textIsBlack ? "!text-black" : "!text-white"}`}>
+        <p className={`font-bold !text-base hidden xs:block ${textIsBlack ? "!text-black" : "!text-white"}`}>
```

Comment:
making it hidden on mobile then enabling block display for anything above `xs` is functionally redunant

Reply:
That's for the typo logo. It affects the responsiveness on really smaller screen, so I hide it and reveal it on a bigger screen with more space.